resotool/create_dummy_resos.py

```diff
@@ -112,6 +112,47 @@
 reso1.save()
 reso1.user_set.add(user1, user2, user3)
 
+recipient1 = Recipient(name='AFD', opening='Hallo,', postal_address='afdhausen')
```

Comment:
In regard of @behrmann's comment on `resolutionemail_detail.html`, all the recipients here should also get an email address to be shown in the table.